src/svelte/components/ListInput.svelte

```diff
@@ -193,7 +193,7 @@
       {:else}
         <!-- svelte-ignore a11y-autofocus -->
         {#if type === 'select'}
-          <svelte:element
+          <select
             this={InputComponent}
```

Comment:
Here and in the other cases, is this still needed? I thought this is what determined what tag was used when using `svelte:element`.

Reply:
You're right. It's not necessary, but I left it there to minimize code changes. I'll remove it (along with some other unnecessary lines), as I've already made more changes than I initially expected.